command_handler.py

```diff
@@ -77,16 +77,11 @@ def validate_paths(dpct_project_path, destination_path):
         des = ""
     else:
         if os.path.exists(args.destination_path):
-            des = '/'+str(args.destination_path)
+            des = '/' + str(args.destination_path)
         else:
             os.mkdir(args.destination_path)  # make directory
 
-    output_folder_path = (str(os.getcwd()) + str(des)+'\outputs').replace('\\','/')
-    if os.path.exists(output_folder_path):
-        pass
-    else:
-        os.mkdir(output_folder_path)  # make directory
-    # print(output_folder_path)
+    output_folder_path = (str(os.getcwd()) + str(des)).replace('\\', '/')
```

Comment:
Paths are presented as different strings on different OS ('\\' for windows vs '/' for mac/linux I believe). Hence, to make it more robust, it's usually the safest option to use a library that handles PosixPath instead of string,  e.g. use Path.joinpath() as in other places